staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -328,12 +328,15 @@ func (s *store) GuaranteedUpdate(
 			// etcd in order to be sure the data in the store is equivalent to
 			// our desired serialization
 			if mustCheckData {
-				origState, err = getCurrentState()
+				origState, err := getCurrentState()
```

Comment:
Don't shadow origState… we want to overwrite it

Reply:
Done.

This is really a bad mistake.